src/components/ActionModal/ActionModal.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { FlatList } from 'react-native';
+import styled, { withTheme } from 'styled-components/native';
+import SlideModal from 'components/Modals/SlideModal';
+import { getThemeColors, themedColors } from 'utils/themes';
+import { MediumText, BaseText } from 'components/Typography';
+import Icon from 'components/Icon';
+import { spacing, fontStyles } from 'utils/variables';
+import type { Theme } from 'models/Theme';
+
+
+type Props = {
+  theme: Theme,
+  onModalClose: (() => void) => void,
+  isVisible: boolean,
+  items: Object[],
+};
+
+const MainContainer = styled.View`
+  padding: 15px ${spacing.layoutSides}px 40px;
+`;
+
+const ItemContainer = styled.TouchableOpacity`
+  flex-direction: row;
+  padding: 15px 0;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Based on designs, those items should be more spaced out. 
Could you please add appropriate amount of vertical padding / margin?
That should also help with current spacing on the top of modal - based on designs it should also be bigger - 30px, not 15px;